packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.html

```diff
@@ -299,10 +299,16 @@ <h4 class="panel-title">
                             data-translate="server_configuration_installation_ovh_stephardraid_label_controllers"
                         >
                         </label>
+                        <span
+                            data-ng-if="informations.hardwareRaid.profile.controllers.length === 1"
+                            data-ng-bind="installation.hardwareRaid.controller.model + ' (' + installation.hardwareRaid.controller.type + ')'"
+                        >
+                        </span>
                         <select
                             class="form-control"
                             id="hardwareRaidController"
                             name="hardwareRaidController"
+                            data-ng-if="informations.hardwareRaid.profile.controllers.length !== 1"
```

Comment:
```suggestion
                            data-ng-if="informations.hardwareRaid.profile.controllers.length > 1"
```
Same for
https://github.com/ovh/manager/pull/8599/files#diff-7f91e120c3b60108dc3b21890db83c0a3c8ad4dd566702a795fc0bbbf1f111e5R352
https://github.com/ovh/manager/pull/8599/files#diff-7f91e120c3b60108dc3b21890db83c0a3c8ad4dd566702a795fc0bbbf1f111e5R375
https://github.com/ovh/manager/pull/8599/files#diff-7f91e120c3b60108dc3b21890db83c0a3c8ad4dd566702a795fc0bbbf1f111e5R691
https://github.com/ovh/manager/pull/8599/files#diff-7f91e120c3b60108dc3b21890db83c0a3c8ad4dd566702a795fc0bbbf1f111e5R966